pl_bolts/models/detection/components/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pl_bolts.models.detection.components.torchvision_backbones import create_torchvision_backbone
```

Comment:
I don't understand why `flake8` doesn't complain about this line without `# noqa: F401`.

Reply:
But why should flake8 complain ?

Reply:
Since it is inlcuded in `__all__`. Flake does not complain